en/docs/scheduled-tasks.md

```diff
@@ -1,23 +1,73 @@
 # Scheduled Tasks
 
-A scheduled task performs a routine task at scheduled time intervals.
+Learn how to execute routine tasks at specified time intervals via scheduled tasks.
 
-e.g., You can configure a scheduled task to publish a summary of GitHub issues (i.e., with details such as the number of GitHub issues in each status) as a message in a group chat of a team every day at a specific time.
+## What is a scheduled task?
 
-The in-built low-code and pro-code editors of Choreo allows you to easily configure scheduled tasks.
+A scheduled task performs a routine task at scheduled time intervals. Following are examples of scenarios where you can schedule tasks:
 
-### Develop
+- Logging a session time-out message for a user at a specific time interval (e.g., every 1 hour).
```

Comment:
Since they are examples, should we make them generic?

Reply:
Removed the time interval given as an example.